search/services/index/util.py

```diff
@@ -16,11 +20,9 @@
 MAX_RESULTS = 10_000
 """This is the maximum result offset for pagination."""
 
-HIGHLIGHT_TAG_OPEN = '<span class="search-hit mathjax">'
-HIGHLIGHT_TAG_CLOSE = '</span>'
-
 SPECIAL_CHARACTERS = ['+', '-', '=', '&&', '||', '>', '<', '!', '(', ')', '{',
                       '}', '[', ']', '^', '~', ':', '\\', '/']
+DEFAULT_SORT = ['_score', '-announced_date_first', '_doc']
```

Comment:
Great change to sort priority. I hope this helps with results.

Curious as to how much feedback about poor results was a result of mirroring old behavior. I suppose we could check on the LANL mirror?